recipes/hyperion/meta.yaml

```diff
@@ -11,6 +11,7 @@ source:
 
 build:
   number: 0
+  skip: True  # [win]
```

Comment:
@astrofrog can you add `[skip appveyor]` in your commits so we can free AppVeyor for the other feedstocks. Unfortunately staged-recipes takes precedence over the feedstocks and the all the Windows stops when there is a build here. (Even when there is no actual like in this case :smile:)